sdk/services/prices.ts

```diff
@@ -139,6 +139,15 @@ export default class PricesService {
 		return this.formatOffChainPrices(pythPrices ?? []);
 	}
 
+	public async getPriceUpdateData(marketKey: FuturesMarketKey) {
```

Comment:
I think this function name could do with being a bit more specific as a public sdk call, maybe `getPythPriceUpdateData`?

Reply:
Good catch, updated this